### PR TITLE
fix: amplify meta file resolution, update yarn.lock

### PR DESCRIPTION
**Pull request by jhockett**
#### Description of changes
- Fixed the logic to handle a non-existant amplify-meta file to match previous behavior that was changed in https://github.com/aws-amplify/amplify-cli/pull/10022
- Building the CLI updated the `yarn.lock` file

#### Checklist

- [x] PR description included
- [ ] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts [hunk 1]
```diff
@@ -460,8 +461,8 @@ export const getDirectiveDefinitions = async (context: $TSContext, resourceDir:
 const s3ResourceAlreadyExists = (): string | undefined => {
   try {
     let resourceName: string;
-    const amplifyMeta = stateManager.getMeta();
-    if (amplifyMeta[AmplifyCategories.STORAGE]) {
+    const amplifyMeta: $TSMeta = stateManager.getMeta(undefined, { throwIfNotExist: false });
```
Comment (lazpavel) [inline packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts:464]: nit: is there a reason we avoid getProjectDetails?

**Review by lazpavel — APPROVED** [review]

**Review by danielleadams — APPROVED** [review]

File: packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts (hunk 1 above)
Comment (jhockett) [reply packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts:464]: Yes, it loads the team-provider-info file even though we are not using it.

File: yarn.lock [hunk 2]
```diff
@@ -7216,6 +7216,13 @@
   resolved "https://registry.npmjs.org/@types/minimist/-/minimist-1.2.2.tgz#ee771e2ba4b3dc5b372935d549fd9617bf345b8c"
   integrity sha512-jhuKLIRrhvCPLqwPcx6INqmKeiA5EWrsCOPhrlFSrbrmU4ZMPjj5Ul/oLCMDO98XRUIwVm78xICz4EPCektzeQ==
 
+"@types/minipass@*":
+  version "3.1.2"
+  resolved "https://registry.npmjs.org/@types/minipass/-/minipass-3.1.2.tgz#e2d7f9df0698aff421dcf145b4fc05b8183b9030"
+  integrity sha512-foLGjgrJkUjLG/o2t2ymlZGEoBNBa/TfoUZ7oCTkOjP1T43UGBJspovJou/l3ZuHvye2ewR5cZNtp2zyWgILMA==
+  dependencies:
+    "@types/node" "*"
+
```
Comment (sundersc) [inline yarn.lock:7225]: Is this change intended?

Comment (jhockett) [reply yarn.lock:7225]: See description, our yarn.lock file was out of sync with our dependencies.
